src/celesto/sdk/runtime/client.py

```
@@ -0,0 +1,747 @@
+    def get(self, end_user_id: str) -> EndUser:
+        """Get an end user's budget and activity.
+
+        Money comes back as :class:`~decimal.Decimal`.
+        """
+        return self._json_request("GET", f"/end_users/{end_user_id}")
+
+    def update(
+        self,
+        end_user_id: str,
+        *,
+        budget_cap_usd: MoneyInput | None | _Unset = UNSET,
```

Ruff (0.16.1)

[warning] `None` not at the end of the type union.

Move `None` to the end of the type union

(RUF036)